### DISCUSSION TITLE
Statistics

### DISCUSSION THREAD

**Discussion by obw-go** [GrandOrgue/grandorgue#1947] [Developers]
Does somebody know where I can find the GrandOrgue development statistics?
At present I'm looking for the number of times files have been downloaded from e.g. release pages...

From the helpful answers I give here the links:
[GitHub Release Stats](https://qwertycube.com/github-release-stats/) (use `grandorgue `in both fields RO & RN)
https://api.github.com/repos/grandorgue/grandorgue (for details)
[Sourceforge for downloads per country](https://sourceforge.net/projects/ourorgan/files/GrandOrgue/stats/map?dates=2023-01-01+to+2023-12-31)

**Comment by Psi58** [comment]
Have you tried https://qwertycube.com/github-release-stats/?
I use it out of curiosity, it's interesting to observe that after the initial burst of downloads following a new release the growth rate is roughly linear. I had expected it to decline as saturation is approached. Perhaps this suggests that a significant number of downloads are by bots, mirrors or archiving services.

**Reply by obw-go** [reply]
@Psi58 
Thx for your response! When I click the link, I have to fill in a depository owner. Do you know who that is?

**Reply by Psi58** [reply]
Just put 'grandorgue' in both boxes...

**Reply by obw-go** [reply]
@Psi58 
Interesting that most users download the Windows version... Is there also information in the GitHub system from wich countries the downloadrequests are coming?

**Reply by Psi58** [reply]
I don't know; I only look at the top-level stats.
If you discover more details please do share them!

**Reply by midimusic** [reply]
Very interesting - I did not know that such information was available for github projects. Is quertycube.com a part of github or are they independent? Does this data originate from github itself?

"Interesting that most users download the Windows version... " This is to be expected as Windows is by far the most popular operating system for personal computers.

You can get a good idea of the countries that are downloading from the Sourceforge website for GO.

csw900

**Reply by obw-go** [reply]
@midimusic 
Thx for your hint looking at [Sourceforge for downloads per country](https://sourceforge.net/projects/ourorgan/files/GrandOrgue/stats/map?dates=2023-01-01+to+2023-12-31).
Good that the recent releases still are offered on Sourceforge as well.

**Reply by Psi58** [reply]
> Is quertycube.com a part of github or are they independent? Does this data originate from github itself?

AFAIK it's nothing to do with Github. I suspect that it does interrogate github for the data, but someone cleverer than me could have a look at the sourcecode (it's on the qwertycube site) to check.

**Comment by midimusic** [comment]
qwertycube is using github's web api to get its results.

You can get a lot of information by using this api. For example just by visiting
https://api.github.com/repos/grandorgue/grandorgue
with your browser produces an amazing quantity of information about grandorgue.

csw900
